Tests/conversionTests.go

```diff
@@ -0,0 +1,20 @@
+package Tests
+
+import (
+	"fmt"
+	"github.com/dosarudaniel/CS438_Project/chord"
+	"strconv"
+)
+
+func main() {
```

Comment:
This code was used to test the `FixFingersDaemon` code.    
`finger[next] = find_successor(n + 2^(next-1))` part